### PR TITLE
Make wl_dir optional, fixes #5

### LINKED ISSUE #5: Should wl_dir be optional?
The comment for `wl_dir` in the README.md seems to suggest that it's also possible to put the files next the locales. I tried omitting the setting, resulting in this error message:
```
$ potypo
Traceback (most recent call last):
  File "/Users/jieter/.virtualenvs/django-tables2/bin/potypo", line 11, in <module>
    sys.exit(main())
  File "/Users/jieter/.virtualenvs/django-tables2/lib/python3.6/site-packages/potypo/__main__.py", line 61, in main
    en_wordlist = Check.get_wordlist(conf['default_language'], conf['wl_dir'], conf['locales_dir'])
  File "/Users/jieter/.pyenv/versions/3.6.3/lib/python3.6/configparser.py", line 1233, in __getitem__
    raise KeyError(key)
KeyError: 'wl_dir'
````

Maybe the setting should be optional?

### PR DESCRIPTION
**Pull request by jieter**

### CODE REVIEW

File: potypo/check.py [hunk 1]
```diff
@@ -42,17 +42,18 @@ def __init__(self, path, wl_dir, chunkers, filters):
...
         file called "<lang>.txt". If this is the case, this should be the
         wordlist we are looking for.
         """
-        for f in os.scandir(wl_dir):
-            if f.is_file() and f.name == lang+".txt":
-                #print("found wordlist in", f.path)
-                return f.path  # as there should be only one language file
+        if wl_dir is not None:
+            wl_dir = os.path.abspath(wl_dir)
+            for f in os.scandir(wl_dir):
```
Comment (jieter) [inline potypo/check.py]: Can't we replace this by something like this:
```
wl_path = os.path.join(wl_dir, lang + '.txt'))
return os.path.isfile(wl_path)
```

File: potypo/__main__.py [hunk 2]
```diff
@@ -43,6 +43,7 @@ def main():
         words = conf['edgecase_words'].strip().split('\n')
         filter_list.append(filters.make_EdgecaseFilter(words))
 
+    wl_dir = conf.get('wl_dir', None)
```
Comment (koebi) [inline potypo/__main__.py]: I'd move this below the errmsg-function, to have the variable declaration as close to its usage as possible.

File: potypo/check.py (hunk 1 above)
Comment (koebi) [reply potypo/check.py]: Yes, but we need to return the path of the file, so this should probably read 
  ``if os.path.isfile(wl_path): return wl_path

**Review by koebi — CHANGES_REQUESTED** [review]
Hey, thank you so much for your PR and your work, I greatly appreciate it :)
I've left two small comments, but apart from that, it looks great :)

**Review by koebi — APPROVED** [review]
Looks good to me, thank you very much :))

**Pull request merged**